src/test/java/com/bookstore/online_bookstore/OnlineBookstoreApplicationTests.java

```diff
@@ -6,8 +6,7 @@
 @SpringBootTest
 class OnlineBookstoreApplicationTests {
 
-	@Test
-	void contextLoads() {
-	}
-
+    @Test
+        void contextLoads() {
```

Comment:
not fixed! do you see any checkstyle exceptions in the console? Try to make some mistakes and build a project with Maven. if you don't see any exceptions check your `pom.xml`.